include/z64player.h

```diff
@@ -875,6 +875,7 @@ typedef struct Player {
 
     /* 0x0850 */ union {
         s16 actionVar2;
+        s16 shakeTimer; // Player_Action_Idle: When set, model y position will oscillate quickly and give the appearnace of shaking.
```

Comment:
appearnace -> appearance

also I've never heard of this, what does it look like in game?

Reply:
I'm pretty sure its unused, but its hard to verify every usage of `actionVar2` with most of them unnamed currently.

https://github.com/user-attachments/assets/1c502b20-b5c5-4cf4-ab00-b13e4e47443b

Reply:
lol, strange

Reply:
I can see it being used as an early version of quake/screen shaking or something. All that really makes sense. But as things become more documented it will be easier to see

Reply:
nah, it's used to animate the player when taking fall damage.

Reply:
oh wow, it is. Did not think think would be handled by the idle action

Reply:
This explains why targeting skips the fall damage animation, something speedrunners have been doing for ages haha. It triggers a different action to be used, even though the timer still gets set.

Pushed the correct usages of `shakeTimer` and removed the "is this used?" comment.